src/test/resources/com/puppycrawl/tools/checkstyle/checks/whitespace/package-info/test1/package-info.java

```diff
@@ -0,0 +1,5 @@
+/*For test allowing to place annotation before PACKAGE_DEF. Should not cause violation.*/
+@Deprecated
+package com.puppycrawl.tools.checkstyle.checks.whitespace.test1;
+
+import java.lang.System;
```

Comment:
please do meaningful imports like at https://github.com/google/guava/blob/master/guava/src/com/google/common/base/package-info.java

the same for other tests where you have import and annotation.